lib/reference/keywords.mjs

```diff
@@ -2,6 +2,7 @@ import uniteSets from '../utils/uniteSets.mjs';
 
 export const basicKeywords = new Set(['initial', 'inherit', 'revert', 'revert-layer', 'unset']);
 
+/** @todo add to {@link fontFamilyKeywords} */
```

Comment:
note

This might be a bug. This needs to be investigated.
i.e. it should be `export const fontFamilyKeywords = uniteSets(basicKeywords, systemFontKeywords, [`
I don't want to change this in this PR.
i.e. `fontFamilyKeywords` is imported by other rules

Reply:
What reason of `const fontFamilyKeywords = uniteSets(basicKeywords, systemFontKeywords)`?

Reply:
Let's discuss that in its own PR or issue.
In short those keywords are accepted by `font-family` too.

Reply:
Okay. At this point, can you remove this comment to avoid any confusion by other people? If this is not a bug, this comment would be unnecessary.

Reply:
I put it here so that it wouldn't be forgotten. Ill remove it.